app/controllers/nominations_controller.rb

```diff
@@ -31,7 +69,7 @@ def receive_school
     if !@nomination_request_form.school.eligible?
       redirect_to not_eligible_nominations_path
     elsif @nomination_request_form.school.fully_registered?
-      redirect_to already_nominated_nominations_path
+      redirect_to already_renominated_nominations_path
```

Comment:
Why change this?

Reply:
He said he changed these to avoid name clashes.